### ISSUE TITLE
Markdown headings

### ISSUE THREAD

**Issue by etbartell** [DS4PS/cpp-526-fall-2019#10]
I can't seem to get the HTML markdown to work with the heading as provided by the lab guide. When I try to knit the document this is the error message I get:

Error in yaml::yaml.load(..., eval.expr = TRUE) : 
  Scanner error: mapping values are not allowed in this context at line 4, column 22
Calls: <Anonymous> ... parse_yaml_front_matter -> yaml_load -> <Anonymous>
Execution halted

I got it to work using the following heading:

```
---
title: "Lab-01-Bartell"
author: "Elliott Bartell"
date: "8/27/2019"
output: html_document
---
```

Is there maybe a package that I need to install for my version of R studio to recognize these additional commands? Thanks!

**Comment by lecy** [comment]
Ok, great. Much easier than package problems !





*Jesse Lecy*
Associate Professor • Arizona State University
Academic Director: M.S. in Program Evaluation and Data Analytics
*M:* +1 801 462 1648 / *@**: *jdlecy@asu.edu / *W**: *publicservice.asu.edu

**Comment by lecy** [comment]
Just curious, had you downloaded the Lab-01.RMD template?

![image](https://user-images.githubusercontent.com/1209099/63896056-d90ff800-c9a5-11e9-9fcf-67f2dcfa25b1.png)

[download link](https://www.dropbox.com/s/43dj0oszgg34clc/Lab-01-LastName.Rmd?dl=1)

**Comment by etbartell** [comment]
Yes I did. I was having some trouble getting R Studio to load the first time so when it finally did I just transcribed it by hand from R. I'm learning the importance of the little details now.

**Comment by lecy** [comment]
It's like a small zen moment brought to you by R. 

Next week's lecture, wax on, and then wax off.

**Comment by castower** [comment]
Hello all,
I downloaded the rmd file directly and I am also getting this same error:

``` {r}
Error in yaml::yaml.load(..., eval.expr = TRUE) : 
  unused argument (eval.expr = TRUE)
Calls: <Anonymous> ... parse_yaml_front_matter -> yaml_load -> <Anonymous>
Execution halted

```

Here is my header:
---
title: 'Lab 01 - Functions and Vectors'
author: "Courtney Stowers"
date: "`r format(Sys.time(), '%d %B, %Y')`"
output:
    html_document:
    df_print: paged
    theme: cerulean
    highlight: haddock
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE, message=F, warning=F)
```

I've tried changing the indents on df_print, theme, and highlight and moving html_document up to the output line, but the error message persists.

Any tips would be great!

Thanks,
Courtney

**Comment by lecy** [comment]
The header is grumpy if you delete spaces. It should be:

```
---
title: 'Lab 01 - Functions and Vectors'
author: "Your Name"
date: "`r format(Sys.time(), '%d %B, %Y')`"
output:
  html_document:
    df_print: paged
    theme: cerulean
    highlight: haddock
---
```

Note: 

```
---
output:
..html_document:         # two spaces
....df_print: paged      # four spaces
....theme: cerulean      # four spaces
....highlight: haddock   # four spaces
---

**Comment by castower** [comment]
I copied and pasted that header from here and I'm still getting the same error message. For some reason, GitHub won't let me upload my rmd file directly, but I uploaded it to GoogleDrive: https://drive.google.com/file/d/1JSXeCz38mXvkG93bHcS42T1_rXEfQvZy/view?usp=sharing

**Comment by lecy** [comment]
Hmm..... not sure. It knit right away for me. 

[Lab-01-Stowers.zip](https://github.com/DS4PS/cpp-526-fall-2019/files/3553996/Lab-01-Stowers.zip)

**Comment by lecy** [comment]
Try closing R Studio (don't save your current session if it asks). Then open again with your lab?

Sometimes you get a ghost in the machine.

**Comment by castower** [comment]
> Try closing R Studio (don't save your current session if it asks). Then open again with your lab?
> 
> Sometimes you get a ghost in the machine.

Following the suggestions here: https://community.rstudio.com/t/error-in-rmarkdown-knit-yaml/21833, I reinstalled the yaml package and now it is working correctly. Thank you!

**Comment by lecy** [comment]
Oh, excellent. I'm glad that worked! 

Package errors are the worst because you keep looking at your code and blaming yourself!
